sycl/unittests/scheduler/CMakeLists.txt

```diff
@@ -17,5 +17,6 @@ add_sycl_unittest(SchedulerTests OBJECT
     RequiredWGSize.cpp
     QueueFlushing.cpp
     PostEnqueueCleanup.cpp
+    Regression.cpp
```

Comment:
Suggest renaming to some descriptive name.

Reply:
I think it will be good to have the separate file for all scheduler regressions similar to the folder https://github.com/intel/llvm/tree/sycl/sycl/test/regression
Does it make sense? In this case the file has name Regression.cpp.
Do you have other ideas regarding the name?

Reply:
the change/fix was to ensure the context was restored when cleaning up when using run-on-host-intel correct?  So maybe instead of "regression.cpp" we should call it "RunOnHostIntelCleanupCorrect.cpp" or similar.